x-pack/legacy/plugins/maps/server/maps_telemetry/test_resources/sample_index_pattern_saved_objects.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
These examples are really complex with a ton of fields. Why not simplify the fields list to something easier to understand, like maybe only one or two fields so its easy to see what purpose they serve in the tests. Then provide a title that makes sense for the test case. So you would have three index patterns with the following titles
1)  indexPattern with geo_point
2) indexPattern with geo_shape
3) indexPattern with no geospatial fields

That way its easier to verify what the test is actually doing 6 months from now